### ISSUE TITLE
PlantUML not working because graphviz not installed

### ISSUE THREAD

**Issue by de-jcup** [de-jcup/eclipse-asciidoctor-editor#47]
Graphviz examples are not rendered as images but contain failure:
```
Failed to generate image: Could not find the 'dot' executable in PATH; add it to the PATH or specify its location using the 'graphvizdot' document attribute
digraph g {
```
Also use cases in plantuml are not rendered correclty, there comes a message 
![image](https://user-images.githubusercontent.com/11644753/39388377-e2b41568-4a7f-11e8-8633-2e4206b0abc2.png)

**Comment by de-jcup** [comment]
This will happen on every machine where plantuml is executed and graphviz is not already installed. PlantUML uses this product to render the images. Its free, see https://graphviz.gitlab.io/license/

So download and install it on your machine: https://www.graphviz.org/download/
_(when .msi installer on a windows server you should install graphviz for all users)_

**Comment by de-jcup** [comment]
Added information also at [FAQ](https://github.com/de-jcup/eclipse-asciidoctor-editor/wiki/FAQ#plantuml-not-working)

**Comment by McMagnus** [comment]
If an image has failed to be generated due to missing GraphViz (or some other issue I guess), it's normally still stored in the plantuml-cache and will be reused even after GraphViz has been installed.

Such failed SVG iages should not be stored in the cache.

**Comment by de-jcup** [comment]
Interesting. But this sounds more like a wanted feature in GraphViz/Plantuml. Maybe you should enter an issue there? With #286 asciidoctor editor will at least drop temp folders/data

**Comment by McMagnus** [comment]
My bad. Didn't check where I ended up after googling. Shall I remove the comment (and this one)?

**Comment by de-jcup** [comment]
@McMagnus No problem, keep it as is. But please do me a favour: If you have created an issue at GraphViz/Plantuml project, it would be nice if  you could add a comment here with the link to the issue inside. Thanx.

**Comment by McMagnus** [comment]
Here's the issue I created on PlantUML.net:

https://forum.plantuml.net/10984/failed-graphviz-graph-is-stored-in-cache

**Comment by anb0s** [comment]
We are facing this problem with own distributions without installed graphviz, but delivered with eclipse e.g. in tools folder.

Would it be possible to configure the path to dot like it's possible in PlantUML plugin also for AsciiDoctor:
![image](https://user-images.githubusercontent.com/95811/95462174-74c8b180-0977-11eb-8135-71dca58aadb1.png)

Should i create new issue and try to do a PR for it?

Thanks!

**Comment by anb0s** [comment]
created new issue #322

**Comment by andressantos10** [comment]
Hey people, did you get any solution for this problem?

**Comment by de-jcup** [comment]
@andressantos10 - this will be addressed with #322 .
